client/webserver/site/src/js/state.ts

```diff
@@ -1,12 +1,23 @@
-const darkModeCK = 'darkMode'
-const authCK = 'dexauth'
-const popupsCK = 'popups'
-const pwKeyCK = 'sessionkey'
-
 // State is a set of static methods for working with the user state. It has
 // utilities for setting and retrieving cookies and storing user configuration
 // to localStorage.
 export default class State {
+  // Cookie keys.
+  static DarkModeCK = 'darkMode'
```

Comment:
We could **almost** move this to local storage. It's only used in the backend template (via `UserInfo.DarkMode`) in two places, both of which could be set with Javascript. I think the reasoning of sending the value from the client is that if you're in dark mode, and you set the CSS class via Javascript, you see a flash of light mode while the script is loading. Maybe there's a way around that, but I feel like it was a thorny issue. 

Anyway, nothing to change here.

Reply:
I guess that's a good example for [my question from above about where to use template/javascript](https://github.com/decred/dcrdex/pull/1975#discussion_r1032065037).